javalib/src/main/scala/java/util/Date.scala

```diff
@@ -35,12 +40,44 @@ class Date(var milliseconds: Long)
   def setTime(time: Long): Unit =
     milliseconds = time
 
-  override def toString(): String = s"Date($milliseconds)"
-
   def toInstant(): Instant = Instant.ofEpochMilli(getTime())
+
+  override def toString(): String = {
+    val seconds = milliseconds / 1000L
+    val default = s"Date($milliseconds)"
```

Comment:
```suggestion
    def default = s"Date($milliseconds)"
```
To make a lazy evaluation in secondsToString work.